src/common/components/witness-card/index.tsx

```diff
@@ -0,0 +1,79 @@
+import moment from "moment";
+import React from "react";
+import { Link } from "react-router-dom";
+import { _t } from "../../i18n";
+import { openInNewSvg } from "../../img/svg";
+import WitnessVoteBtn from "../witness-vote-btn";
+
+export const WitnessCard = ({
+  voted,
+  row,
+  witness,
+  onSuccess,
+  ...other
+}: any) => {
+  return (
+    <div className="p-3 mb-3 border rounded">
+      <div className="d-flex align-items-center justify-content-between">
+        <div>
+          <h3>
+            <Link to={`/@${row.name}`}><b>{row.name}</b></Link>
+            {row.witnessBy && (
+              <div className={"notranslate"}>
+                <small>by {row.witnessBy}</small>
+              </div>
+            )}
+          </h3>
+        </div>
+        <div>
+          <img
+            src={`https://images.ecency.com/webp/u/${row.name}/avatar/medium`}
```

Comment:
This should check if webp is supported or not, please use global.canusewebp check